dirty-flag/src/main/java/com/iluwatar/dirtyflag/World.java

```diff
@@ -0,0 +1,47 @@
+package com.iluwatar.dirtyflag;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * 
+ * A middle-layer app that calls/passes along data from the back-end.
+ * 
+ * @author swaisuan
+ *
+ */
+public class World {
+
+  private static World world;
+  private static List<String> countries = new ArrayList<String>();
+
+  private World() {
+  }
+
+  /**
+   * Init.
+   * 
+   * @return World instance
+   */
+  public static World getInstance() {
+    if (world == null) {
```

Comment:
Suggestion: Just document a warning that this is not a thread safe way to implement Singleton. Refer to Singleton for more information.